pkg/registry/core/service/ipallocator/controller/repairip.go

```diff
@@ -21,7 +21,6 @@ import (
 	"fmt"
```

Comment:
it seems like `"move repair loop to controller"` ought to be squashed with `"add repair loop"`?

Reply:
yes, you are right